src/embed.js

```diff
@@ -229,4 +233,30 @@ function honeyAddy() {
   }
 }
 
-module.exports = { welcomeEmbed, brightidWarningEmbed, wrongChannelWarningEmbed, helpEmbed, pollenEmbed, brightidEmbed, honeyAddy }
+function credEmbed(totalCred, length, cred) {
+  return {
+    embed: {
+      color: 16769024,
+      title: 'Your cred:',
+      thumbnail: {
+        url: 'https://i.imgur.com/dRxtULu.png',
+      },
+      fields: [
+        {
+          name: 'Total',
+          value: Math.round(totalCred)
+        },
+        {
+          name: 'Last week',
+          value: Math.round(cred[length - 1])
+        },
+      ],
+      timestamp: new Date(),
+      footer: {
+        text: 'about.1hive.org',
```

Comment:
```suggestion
        text: '1hive.github.io/pollen',
```

Maybe we can put the link to the site in the footer? what do you think